qbittools/utils.py

```diff
@@ -0,0 +1,41 @@
+def format_bytes(size):
+    power = 2**10
+    n = 0
+    power_labels = {0: "B", 1: "KiB", 2: "MiB", 3: "GiB", 4: "TiB"}
+    while size > power:
+        size /= power
+        n += 1
+    formatted = round(size, 2)
+    return f"{formatted} {power_labels[n]}"
+
+
+def filter_tracker_by_domain(domain):
+    import qbittools
```

Comment:
Would it be better to the config list of trackers as an function arg to `filter_tracker_by_domain` ? This import will happen on every iteration instead of just once.